### PR TITLE
(HOME-2161) add defaults to schema fields

### PR DESCRIPTION
**Pull request by kevinbrosamle**

### CODE REVIEW

File: api/v3/schema/consumer.res.json [hunk 1]
```diff
@@ -3,105 +3,143 @@
...
+							"title": "Portfolio Section Title",
+							"description": "Landing, Middle Right Quadrant",
+							"default": "A Customized Portfolio For You"
 						},
 						"paragraphs": {
 							"type": "array",
-							"description": "Portfolio Section Text, Landing, Middle Right Quadrant",
+							"title": "Portfolio Section Text",
+							"description": "Landing, Middle Right Quadrant",
+							"default": [],
```
Comment (dancrogers) [inline api/v3/schema/consumer.res.json]: duplicate key here

File: api/v3/schema/consumer.res.json [hunk 2]
```diff
@@ -3,105 +3,143 @@
...
+			"title": "Address",
 			"description": "Address",
 			"properties": {
-				"zipCode": {
+				"line1": {
 					"type": ["string", "null"],
-					"description": "Zip Code"
+					"title": "Address Line 1",
+					"description": "Address Line 1",
+					"default": "Address Line 1"
```
Comment (antoinemeyer) [inline api/v3/schema/consumer.res.json]: Does a default makes sense for those fields?

Comment (kevinbrosamle) [inline api/v3/schema/consumer.res.json]: yeah guess it wouldn't, removing

File: api/v3/schema/consumer.res.json [hunk 3]
```diff
@@ -110,111 +147,157 @@
...
 					"type": "object",
+					"title": "Transfer Section",
 					"description": "Transfer Section",
 					"properties": {
 						"toAccountName": {
 							"type": ["string", "null"],
-							"description": "Transfer Section Account Name - Landing, Middle Left Quadrant"
+							"title": "Transfer Section Account Name",
+							"description": "Landing, Middle Left Quadrant",
+							"default": "Checking ...2546"
```
Comment (dancrogers) [inline api/v3/schema/consumer.res.json]: Probably should be empty defaults for toAccountName and symbol

File: api/v3/schema/consumer.res.json [hunk 4]
```diff
@@ -110,111 +147,157 @@
...
+							"default": [],
 							"items": {
 								"type": "object",
+								"title": "Portfolio Holdings Item",
 								"description": "Portfolio Holdings Item",
 								"properties": {
 									"symbol": {
 										"type": ["string", "null"],
-										"description": "Symbol"
+										"title": "Symbol",
```
Comment (dancrogers) [inline api/v3/schema/consumer.res.json]: Blank defaults for symbol, percentage, description

File: api/v3/schema/consumer.res.json [hunk 5]
```diff
@@ -110,111 +147,157 @@
...
 									}
 								}
 							}
 						},
 						"portfolioName": {
 							"type": ["string", "null"],
-							"description": "Name Of Portfolio"
+							"title": "Portfolio Name",
+							"description": "Name Of Portfolio",
+							"default": "Portfolio Name"
```
Comment (dancrogers) [inline api/v3/schema/consumer.res.json]: Either blank default or something generic like "My Portfolio" or just "Portfolio"

File: api/v3/schema/consumer.res.json [hunk 6]
```diff
@@ -227,170 +310,190 @@
...
 					"items": {
 						"$ref": "#/definitions/address"
 					}
 				},
 				"phone": {
 					"type": ["string", "null"],
-					"description": "Firm Phone Number"
+					"title": "Firm Phone Number",
+					"description": "Firm Phone Number",
+					"default": "123-456-7890"
```
Comment (dancrogers) [inline api/v3/schema/consumer.res.json]: probably should be "" for phone and email

File: api/v3/schema/consumer.res.json [hunk 7]
```diff
@@ -227,170 +310,190 @@
...
+					"title": "Firm Email",
+					"description": "Firm Email",
+					"default": "email@email.com"
 				}
 			}
 		},
 		"firmName": {
 			"type": ["string", "null"],
-			"description": "Firm Name - Landing/Footer/Authorization/MyPortfolio"
+			"title": "Firm Name",
```
Comment (dancrogers) [inline api/v3/schema/consumer.res.json]: Same for Firm name, Embeddable Video, Product name.

File: api/v3/schema/consumer.res.json [hunk 8]
```diff
@@ -227,170 +310,190 @@
...
 					"type": "object",
+					"title": "About Us",
 					"description": "About Us",
 					"properties": {
 						"text": {
 							"type": ["string", "null"],
-							"description": "About Us Text, About Us"
+							"title": "About Us Text",
+							"description": "About Us",
+							"default": "At Investitall we have years of experience helping people achieve their financial goal-whether saving for college or planning for retirement. Our seasoned investment team is versed in modern portfolio theory and sophisticated investment management techniques so you know your portfolio is in good hands. We'll focus on your investments so you can focus on the things that matter most to you."
```
Comment (dancrogers) [inline api/v3/schema/consumer.res.json]: Maybe remove "At Investitall" and start we "We have years..."

File: api/v3/schema/consumer.res.json [hunk 9]
```diff
@@ -227,170 +310,190 @@
...
 								"type": "object",
+								"title": "Person",
 								"description": "Person",
 								"properties": {
 									"jobTitle": {
 										"type": ["string", "null"],
-										"description": "Job Title"
+										"title": "Job Title",
+										"description": "Job Title",
+										"default": "Job Title"
```
Comment (dancrogers) [inline api/v3/schema/consumer.res.json]: "" Job title and name

File: api/v3/schema/consumer.res.json [hunk 10]
```diff
@@ -400,59 +503,82 @@
...
 								"type": "object",
+								"title": "Section",
 								"description": "Section",
 								"properties": {
 									"title": {
 										"type": ["string", "null"],
-										"description": "Title"
+										"title": "Title",
+										"description": "Title",
+										"default": "Title"
```
Comment (dancrogers) [inline api/v3/schema/consumer.res.json]: Same here

File: api/v3/schema/consumer.res.json [hunk 11]
```diff
@@ -400,59 +503,82 @@
...
 								"type": "object",
+								"title": "Items",
 								"description": "Items",
 								"properties": {
 									"title": {
 										"type": ["string", "null"],
-										"description": "Title"
+										"title": "Title",
+										"description": "Title",
+										"default": "Title"
```
Comment (dancrogers) [inline api/v3/schema/consumer.res.json]: ""

File: api/v3/schema/consumer.res.json [hunk 12]
```diff
@@ -461,21 +587,30 @@
...
 								"type": "object",
+								"title": "Section",
 								"description": "Section",
 								"properties": {
 									"title": {
 										"type": ["string", "null"],
-										"description": "Title"
+										"title": "Title",
+										"description": "Title",
+										"default": "Title"
```
Comment (dancrogers) [inline api/v3/schema/consumer.res.json]: ""

File: api/v3/schema/consumer.res.json (hunk 3 above)
Comment (kevinbrosamle) [inline api/v3/schema/consumer.res.json]: gonna leave blank, but actually have a default in the UI at the moment so will use that

File: api/v3/schema/consumer.res.json (hunk 4 above)
Comment (kevinbrosamle) [inline api/v3/schema/consumer.res.json]: gonna leave blank for now until I talk with denise about what she wants as default here

File: api/v3/schema/consumer.res.json [hunk 13]
```diff
@@ -3,105 +3,136 @@
...
-					"description": "Landing Images",
+					"title": "Landing Page Images",
+					"description": "Images - Choose file at top of page",
 					"properties": {
 						"everydayBg": {
 							"type": ["string", "null"],
-							"description": "Image - Landing, Bottom Left Quadrant",
+							"title": "Everyday BG Image",
+							"description": "Landing, Bottom Left Quadrant",
+							"default": "https://s3-us-west-1.amazonaws.com/trizic-investitall-public/firms/2/S3_EVERYDAY_BG_IMAGE/Background.png",
```
Comment (antoinemeyer) [inline api/v3/schema/consumer.res.json]: Do we really have defaults for images?
The resource that is linked refers to firm with id 2 on environment investitall. Not sure that is a great default.

Comment (kevinbrosamle) [inline api/v3/schema/consumer.res.json]: talked to @gregdurham about using investitall defaults. i think its ok, or do you think we should just leave the image blank?

Comment (antoinemeyer) [inline api/v3/schema/consumer.res.json]: either blank images or static images hosted somewhere where it's not environment dependant.
imagine tomorrow someone at CS deletes those images on investitall, impacting all firms using that default in use01.

Comment (kevinbrosamle) [inline api/v3/schema/consumer.res.json]: ok ill set to blank for now

**Pull request merged**